demo/ota_demo.c

```diff
@@ -30,8 +30,7 @@ static uint32_t totalBytesReceived = 0;
 static uint8_t downloadedData[ CONFIG_MAX_FILE_SIZE ] = { 0 };
 char globalJobId[ MAX_JOB_ID_LENGTH ] = { 0 };
 
-static void handleMqttStreamsBlockArrivedCallback(
-    MqttFileDownloaderDataBlockInfo_t * dataBlock );
```

Comment:
Can we now get rid of the `MqttFileDownloaderDataBlockInfo_t` struct? And if not, why swap it for it's components instead?